python/kserve/kserve/model.py

```diff
@@ -59,7 +59,9 @@ def get_latency_ms(start: float, end: float) -> float:
 
 
 class Model:
-    def __init__(self, name: str):
+    def __init__(self, name: str, predictor_host: str = None,
+                 predictor_protocol: str = PredictorProtocol.REST_V1.value,
+                 predictor_use_ssl: bool = False, predictor_request_timeout_seconds: int = 600):
```

Comment:
How about extracting this into a `predictor_config: PredictorConfig` arg instead?